source/developer/mobile-developer-setup.rst

```diff
@@ -141,14 +130,14 @@ If you need to add a new dependency to the project, it is important to add them
 JavaScript Only
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you need to add a new JavaScript dependency that is not related to React Native, **use yarn, not npm**. Be sure to save the exact version number to avoid conflicts in the future. 
+If you need to add a new JavaScript dependency that is not related to React Native, **use npm, not yarn**. Be sure to save the exact version number to avoid conflicts in the future. 
 
-	e.g. ``yarn add -E <package-name>``
+	e.g. ``npm add -E <package-name>``
```

Comment:
this command does not exist in npm, please update

Reply:
That command works fine for me